packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -125,6 +66,10 @@ export default class extends EntityModel {
     const entityModel = await EntityModel.getLatest(graphApi, {
       accountId: workspaceAccountId,
       entityId,
+    }).catch((err: AxiosError) => {
+      throw new Error(
+        `failed to get user entity with id ${entityId}: ${err.code} ${err.response?.data}`,
```

Comment:
We don't really do this anywhere else, I'm happy to delete it (especially because data isn't always there) but this is generally the problem I've been encountering where panics aren't giving the right information.

Reply:
Have responded to that :) Feel free to resolve this if you're happy with the suggestion

Reply:
Yeah I didn't want to throw in a todo in this particular case. Thanks!